temporal-sdk/src/test/java/io/temporal/internal/testing/ActivityTestingTest.java

```diff
@@ -68,7 +75,7 @@ public String activity1(String input) {
   @Test
   public void testSuccess() {
     testEnvironment.registerActivitiesImplementations(new ActivityImpl());
-    TestActivity activity = testEnvironment.newActivityStub(TestActivity.class);
+    TestActivity activity = testEnvironment.newActivityStub(TestActivity.class, activityOptions);
```

Comment:
I don't understand why we are making these changes here. I think we should add the ability to set ActivityOptions on the default TestActivityEnvironment.

Reply:
l thought it to be misleading to have different behavior in prod and in tests. If you don't set required timeouts in the stub in prod and expect things to break, this is what should happen in tests. Consistent behavior is important for testing for all coming engineers who would have to learn the product.
Alternatively we can rename the method to something like TestActivityEnvironment#newActivityStubWithDefaults which would hopefully help with confusion.

Reply:
I think for TestActivityEnvironment we can provide default ActivityOptions with a large timeout.